.sync.yml

```diff
@@ -13,7 +13,11 @@
     - /provision.yaml
 Rakefile:
   default_disabled_lint_checks:
-    - '140chars'
+    - relative
```

Comment:
mhm do we need to disable the `relative` check?

Reply:
not sure why that was there, removed...